### PR TITLE
Fix CFLAGS for Clang

### PR DESCRIPTION
**Pull request by paulmenzel**

### CODE REVIEW

**Comment by davidcunado-arm** [comment]
jenkins: test this please

File: Makefile [hunk 1]
```diff
@@ -131,13 +131,13 @@ TF_CFLAGS_aarch32	=	-target arm-arm-none-eabi -march=armv8-a
 TF_CFLAGS_aarch64	=	-target aarch64-arm-none-eabi -march=armv8-a
 else ifneq ($(findstring clang,$(notdir $(CC))),)
 TF_CFLAGS_aarch32	=	-target armv8a-none-eabi
-TF_CFLAGS_aarch64	=	-target aarch64-elf
+TF_CFLAGS_aarch64	=	-target aarch64-elf -mstrict-align
```
Comment (dp-arm) [inline Makefile]: Just realized, shouldn't this be in the else clause on line 137?  You are adding -mstrict-align for the clang configuration.

**Comment by dp-arm** [comment]
@paulmenzel ping

**Comment by davidcunado-arm** [comment]
@paulmenzel 
I'm going to close this PR since there's not been any traction on it. Please re-open if you have an update available. Thanks!

**Pull request closed without merging**